### DISCUSSION TITLE
Floorplan plugin

### DISCUSSION THREAD

**Discussion by Azmodeszer** [netbox-community/netbox#15298] [Q&A] [answered]
I know this is about a plugin and not NB itself, but where to post this is part of the issue itself...

So the floorplan plugin is listed as part of the official community plugins: https://netboxlabs.com/netbox-plugins/

I'm having an issue with it I posted on the original repository the netboxlabs one was forked from: https://github.com/tbotnz/netbox_floorplan/issues/26 but by all appearances there hasn't been any activity for weeks, if not months. Then I tried the netboxlabs fork and got the same error. I guess better to go with the official fork, but since that one does not have issues enabled, I'm wondering where I should post mine now? Is it okay to just post issues pertaining to official/certified plugins here?

**Comment by jeremystretch** [comment]
No; you'll need to submit issues with _any_ plugins on their respective repos. This repo accepts only issues pertaining to NetBox itself.

**Reply by Azmodeszer** [reply]
Fair enough. Guess I'll have to wait and see if the tbotnz rep returns to activity at some point, since I cannot post issues in the fork.

**Comment by jcollie** [comment]
> No; you'll need to submit issues with _any_ plugins on their respective repos. This repo accepts only issues pertaining to NetBox itself.

That seems pretty disingenuous when https://github.com/netboxlabs/netbox-floorplan-plugin has issues disabled.

**Reply by DanSheps** [reply]
That is a mirror of https://github.com/tbotnz/netbox_floorplan, which looks to be for internal use.

**Comment by jeremystretch** [comment]
> That seems pretty disingenuous when https://github.com/netboxlabs/netbox-floorplan-plugin has issues disabled.

Would you submit a NetBox bug report to the Django project? Of course not. The relationship between NetBox and a NetBox plugin is no different.

**Comment by jeffgdotorg** [comment, marked as answer]
@Azmodeszer @jcollie I'm really sorry about the problems you're having with the Floorplan plugin, and I'm working to set things right. Thanks for bearing with us as we build the plane in which you're attempting to fly.

In the interest of clarifying and setting expectations:

- I signed off on marking Floorplan as certified / supported because I know that NetBox Labs staff have access to the levers they need to make the plugin as good as it ought to be.
- We will soon (think days rather than weeks) move a canonical fork of Floorplan to the `netbox-community` org and enable issues and all the other collaborative features you expect. I believe one of our internal teams has customer-facing processes that depend on the fork in the `netboxlabs` org, and I want to be sure that we don't throw sand into those gears.

Apologies for the sand that has fallen into your gears in the meantime. Non-customer users are every bit as important to NetBox Labs' success as our customers are, but I've done a poor job *showing* that in this case.

**Reply by Azmodeszer** [reply]
Cheers, glad to hear it! No worries.

**Reply by jeffgdotorg** [reply]
Yesterday we moved the Floorplan plugin's repo to https://github.com/netbox-community/netbox-floorplan-plugin. Issues are now enabled, so please recreate your issue there.

Earlier this week I spoke with @tbotnz and we agreed the fork linked above will be the "fork of record" going forward, with Tony's original repo living on as a record of the plugin's early days. Updates to that repo's README will soon explain the situation and redirect visitors.
